packages/graphql-cli-validate-queries/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
It would be good to have feedback on this license. Should it be the same as PWA Studio?  Is the `Copyright (c) 2019 Adobe Inc.` correct?

Reply:
Yeah, this is fine. We have broad approval for this license, and the graphql-cli tool doesn't stipulate anything about the licenses of its plugins.